src/main/java/jp/co/metateam/library/controller/StockController.java

```diff
@@ -138,10 +138,10 @@ public String update(@PathVariable("id") String id, @Valid @ModelAttribute Stock
 
     /**
      * 在庫カレンダー画面初期表示処理
-     * @param year
+     * @param Integer year Viewから受け取った値。nullも可。
```

Comment:
FYI :
```suggestion
     * @param year Integer Viewから受け取った値。nullも可。
```
javaの場合は[変数名 型 説明]の方がよいみたいです!!